WTWSMS/decisions/WtWSMS_religious_decisions.txt

```diff
@@ -407,57 +407,72 @@ decisions = {
 	create_priscillian_patriarchy = {
 		only_playable = yes
 		is_high_prio = yes
-	
 		potential = {
 			religion = priscillianism
 			NOT = { is_title_active = d_priscillianism }
 			age = 16
 			prisoner = no
 			NOT = { trait = incapable }
 		}
-		
 		allow = {
-			is_heretic = no
 			piety = 1000
+			trait = vegetarian
+			martial = 8
+			learning = 8
```

Comment:
A ruler who has militarily managed to defeat all the others and manages to be at the verge of restoration to this shouldn't be hindered because he isn't Vegetarian or lacks one stat. These are hard-restrictions because if you are a 0 ruler you have no possibility of reaching these values.